### ISSUE TITLE
pretrained model: number of epochs?

### ISSUE THREAD

**Issue by trougnouf** [PingoLH/FCHarDNet#2]
What is the number of epochs for your pretrained model?
Are the parameters those listed in your config files and default args?
Thank you.

**Comment by PingoLH** [comment]
Hello, thanks for asking. Yes, the hyperparameters for training FCHarDNet-70 were all listed in config/hardnet.yml. You can also add color jittering augmentation to improve test-set accuracy. 
For the ImageNet pretrained model, it was trained with lr=0.1, batch size=512, weight decay=1e-4. The training time was a little bit tricky: Max epoch=150, but early stopped at epoch 110~120, since I noticed that there is no big difference for FCHarDNet training with a base model that is fully pretrained or not.

**Comment by trougnouf** [comment]
Thank you @PingoLH  !
